src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -937,6 +937,17 @@ public function register(string $id, ?string $class = null): Definition
         return $this->setDefinition($id, new Definition($class));
     }
 
+    /**
+     * Registers a service definition.
+     *
+     * This method allows for simple registration of service definition
+     * with a fluid interface.
+     */
```

Comment:
This docblock is c&p'ed from the previous method. We should adjust it a little because the two methods do different things, don't they.

Reply:
Good point, updated.